lib/src/kbasesearchengine/main/NarrativeInfoDecorator.java

```diff
@@ -44,6 +46,8 @@
     private final SearchInterface searchInterface;
     private final NarrativeInfoProvider narrInfoProvider;
     private final AuthInfoProvider authInfoProvider;
+    private final String removedGuids = "removedGuids";
```

Comment:
All the other API properties are under_score rather than camelCase, so this one should be as well